blueflood-http/src/main/java/com/rackspacecloud/blueflood/outputs/handlers/HttpMultiRollupsQueryHandler.java

```diff
@@ -120,10 +124,13 @@ public void handle(ChannelHandlerContext ctx, HttpRequest request) {
             final String jsonStringRep = gson.toJson(element);
             sendResponse(ctx, request, jsonStringRep, HttpResponseStatus.OK);
         } catch (InvalidRequestException e) {
+            log.debug(e.getMessage());
             sendResponse(ctx, request, e.getMessage(), HttpResponseStatus.BAD_REQUEST);
```

Comment:
did you purposely leave out the exception parameter on this line?  you include it on the others.